### PR TITLE
Feature/rename scorez sparkz

### PR DESCRIPTION
**Pull request by i-skrypnyk**

### CODE REVIEW

File: src/main/scala/sparkz/core/app/Application.scala [hunk 1]
```diff
@@ -75,7 +75,7 @@ trait Application extends ScorexLogging {
     }
   }
 
-  val scorexContext = ScorexContext(
+  val scorexContext = SparkzContext(
```
Comment (paologalligit) [inline src/main/scala/sparkz/core/app/Application.scala]: Can we rename it too as `sparkzContext`?

File: src/main/scala/sparkz/core/network/NetworkController.scala [hunk 2]
```diff
@@ -29,7 +31,7 @@ import scala.util.Try
   */
 class NetworkController(settings: NetworkSettings,
                         peerManagerRef: ActorRef,
-                        scorexContext: ScorexContext,
+                        scorexContext: SparkzContext,
```
Comment (paologalligit) [inline src/main/scala/sparkz/core/network/NetworkController.scala]: We can rename the variable name to `sparkzContext` and the other four occurrences below

File: src/main/scala/sparkz/core/network/PeerConnectionHandler.scala [hunk 3]
```diff
@@ -22,7 +22,7 @@ import scala.util.{Failure, Success}
 class PeerConnectionHandler(val settings: NetworkSettings,
                             networkControllerRef: ActorRef,
                             peerManagerRef: ActorRef,
-                            scorexContext: ScorexContext,
+                            scorexContext: SparkzContext,
```
Comment (paologalligit) [inline src/main/scala/sparkz/core/network/PeerConnectionHandler.scala]: Same as above with the other occurrences in this source

File: src/main/scala/sparkz/core/network/peer/PeerManager.scala [hunk 4]
```diff
@@ -14,7 +14,7 @@ import scala.util.Random
   * Peer manager takes care of peers connected and in process, and also chooses a random peer to connect
   * Must be singleton
   */
-class PeerManager(settings: ScorexSettings, scorexContext: ScorexContext) extends Actor with ScorexLogging {
+class PeerManager(settings: SparkzSettings, scorexContext: SparkzContext) extends Actor with ScorexLogging {
```
Comment (paologalligit) [inline src/main/scala/sparkz/core/network/peer/PeerManager.scala]: Same here

File: src/test/scala/sparkz/core/network/peer/PeerManagerSpec.scala [hunk 5]
```diff
@@ -22,7 +22,7 @@ class PeerManagerSpec extends NetworkTests {
 
 
     val selfAddress = settings.network.bindAddress
-    val scorexContext = ScorexContext(Seq.empty, Seq.empty, None, timeProvider, Some(selfAddress))
+    val scorexContext = SparkzContext(Seq.empty, Seq.empty, None, timeProvider, Some(selfAddress))
```
Comment (paologalligit) [inline src/test/scala/sparkz/core/network/peer/PeerManagerSpec.scala]: Same here

**Review by paologalligit — CHANGES_REQUESTED** [review]
Looks good, just some variable renaming and this sbt file too https://github.com/HorizenOfficial/Sparkz/blob/feature/rename_scorez_sparkz/testkit/build.sbt#L1 and I think it's good to go

File: src/main/scala/sparkz/core/app/Application.scala (hunk 1 above)
Comment (i-skrypnyk) [reply src/main/scala/sparkz/core/app/Application.scala]: Done, fixed all of these, thanks!

**Review by paologalligit — APPROVED** [review]

**Review by i-Alex — CHANGES_REQUESTED** [review]
Let's change also the `testkit/README.md` file. scorex-testkit -> sparkz-testkit

**Review by i-Alex — APPROVED** [review]

**Pull request merged**